packages/wallet/frontend/src/pages/transactions.tsx

```diff
@@ -290,7 +290,7 @@ const TransactionsPage: NextPageWithLayout<TransactionsPageProps> = ({
                         trx.walletAddressUrl ??
                         ''}
                       {trx.walletAddressUrl ? (
-                        <span className="tooltip -ml-10 -mt-11 rounded border border-turqoise bg-white p-2 text-base shadow-lg">
+                        <span className="tooltip -ml-10 -mt-11 rounded-md bg-green-dark dark:bg-purple-bright p-2 text-white shadow-lg">
```

Comment:
Can you add a todo item here to replace the tooltip with the one from Radix UI (just like we did for the account details page).